.github/CONTRIBUTING.md

```diff
@@ -49,3 +49,70 @@ cd $HOME/go-vela/mock
 # Add a remote branch pointing to your fork
 git remote add fork https://github.com/your_fork/mock
 ```
+
+### Development
+
+* Navigate to the repository code:
+
+```bash
+# Change into the project directory
+cd $HOME/go-vela/mock
+```
+
+* Write your code
+  * Please be sure to [follow our commit rules](https://chris.beams.io/posts/git-commit/#seven-rules)
+  * Please address linter warnings appropriately. If you are intentionally violating a rule that triggers a linter, please annotate the respective code with `nolint` declarations [[docs](https://golangci-lint.run/usage/false-positives/)]. we are using the following format for `nolint` declarations:
+
+    ```go
+    // nolint:<linter(s)> // <short reason>
+    ```
+  
+    Example:
+
+    ```go
+    // nolint:gocyclo // legacy function is complex, needs simplification
+    func superComplexFunction() error {
+      // ..
+    }
+    ```
+
+    Check the [documentation for more examples](https://golangci-lint.run/usage/false-positives/).
+
+* Ensure your code meets the project standards:
+
+```bash
+# Clean the code with `go`
+go mod tidy
+go fmt ./...
+go vet ./...
```

Comment:
Yea, I started all this on the `/types` repo to use as a template. I should have asked about it there because I had the same thought and then I assumed that we are doing it on purpose to show how to do it without `make`. I'm somewhat indifferent as long as it's consistent.

Reply:
Fair enough 👍 

I'll admit that I lean towards `make clean` but only slightly!

I prefer `make clean` because it grants us the flexibility to choose how much or little get's ran based off that command.

For example, if we decide to add other elements to that procedure, (more `go` based commands, maybe dep checks, maybe linting checks etc.) the procedure for running those from a user perspective will still always be `make clean` but we get the benefit of choosing what logic get's executed off that command.

Perhaps we'll let others chime in with their thoughts?

Reply:
ok, applied a fix and actually matched a bit more to recent changes in server contributing docs. thanks for the feedback. i will apply the same in `types` in an upcoming PR.